react/index.js

```diff
@@ -1,27 +1,63 @@
-import React from 'react'
+import React, {Component} from 'react'
 import PropTypes from 'prop-types'
-import {graphql} from 'react-apollo'
-import query from './query.graphql'
-
-const Library = ({data: {loading, books, sampleStaticResolver}}) => {
-  return (
-    <div>
-      <h3>My Library</h3>
-      {loading && 'loading'}
-      {!loading && books.map(({id, name, author}) => <div key={id}>{name} - {author}</div>)}
-      {!loading && sampleStaticResolver.result}
-    </div>
-  )
+import {compose, graphql} from 'react-apollo'
+
+import createAutopersistedBook from './graphql/createAutopersistedBook.graphql'
+import deleteAutopersistedBook from './graphql/deleteAutopersistedBook.graphql'
+import createCachedBook from './graphql/createCachedBook.graphql'
+import deleteCachedBook from './graphql/deleteCachedBook.graphql'
+import getBooks from './graphql/getBooks.graphql'
+
+import {BooksManager} from './booksManager'
+
+class Library extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const {
+      data: {autopersistedBooks, cachedBooks},
+      createAutopersistedBook,
+      deleteAutopersistedBook,
+      createCachedBook,
+      deleteCachedBook
+    } = this.props
+
+    const inlineStyle = {
+      padding: '15px'
+    }
+
+    return (
+      <div style={inlineStyle}>
+        <BooksManager
+          title="Auto Persisted Books"
+          books={autopersistedBooks}
+          onCreate={createAutopersistedBook}
+          onDelete={deleteAutopersistedBook}
+          onGet={getBooks}
+        />
+        <BooksManager
+          title="Cached Books"
+          books={cachedBooks}
+          onCreate={createCachedBook}
+          onDelete={deleteCachedBook}
+          onGet={getBooks}
+        />
+      </div>
+    )
+  }
 }
 
 Library.propTypes = {
   data: PropTypes.object,
+  mutate: PropTypes.func
 }
 
-export default graphql(query, {
-  options: {
-    variables: {
-      param: "hardcoded",
-    }
-  }
-})(Library)
+export default compose(
+  graphql(createAutopersistedBook, {name: 'createAutopersistedBook'}),
```

Comment:
There's no need to name each mutation here. When you don't name them, they'll already have these exact same names, but in the `mutation` prop. Works either way, but I think it's cleaner to just use the destructuring with the default options here. You could try it out and see if you prefer it as well :)